src/main/kotlin/org/cascadebot/cascadebot/commandmeta/CommandContext.kt

```diff
@@ -141,6 +143,10 @@ class CommandContext(
         }
     }
 
+    fun replyUsage() {
+        uiMessaging.replyUsage()
+    }
+
```

Comment:
Just a method that proxies another message?

Reply:
Thanks for pointing this out. I thought more used the other usage method in context so I added this as a replacement, but not a lot of other do.